api/add_compression_policy.md

```diff
@@ -60,6 +61,12 @@ Add a policy to compress chunks older than 60 days on the `cpu` hypertable.
 SELECT add_compression_policy('cpu', INTERVAL '60d');
 ```
 
+Add a policy to compress chunks created 3 months before on the 'cpu' hypertable.
+
+``` sql
+SELECT add_compression_policy('cpu', compress_created_before => INTERVAL '3 months');
+```
+
```

Comment:
I think you need to elaborate on this a little. For example, if I have a chunk interval of 1 month and specify a `compress_created_before` of three months, what is the most recent data that will be compressed? It can be computed using the information here, but it would be good to give some more context to make the text a little easier to work with.

Reply:
Yes, but it would be better with some more context just to make it easier to understand. (I was thinking about the situation where the current data is inserted and hence the creation time will be the same as the insertion time of the first row.)